data-es6.js

```diff
@@ -17,6 +17,16 @@ exports.tests = [
   category: 'optimisation',
   significance: 'medium',
   spec: 'http://www.ecma-international.org/ecma-262/6.0/#sec-tail-position-calls',
+  note_id: 'proper-tail-call-optimisation',
```

Comment:
this feature is called "proper tail calls". It is not an optimization and must not be described as one.
```suggestion
  note_id: 'proper-tail-calls',
```